package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "innlevering",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "cd server && npm start",
+    "dev": "concurrently npm:dev:client npm:dev:server",
+    "dev:client": "cd client && npm run dev",
+    "dev:server": "cd server && npm run dev",
+    "format": "prettier --write **/*.{js,jsx}",
+    "format:check": "prettier --check **/*.{js,jsx}",
+    "build": "npm run build:server && npm run build:client",
+    "build:client": "cd client && npm run build",
+    "build:server": "cd server && npm run build",
+    "test": "jest",
```

Comment:
Det er vrient å få testene både for React og express å kjøre fra rot-prosjektet. Det kan være enklere å kjøre testene separat for disse to prosjektene.